src/Search/Search.jsx

```diff
@@ -131,3 +139,8 @@ export default class Search extends Component {
     )
   }
 }
+
+
+Search.defaultProps = {
+  NoResultsComponent: () => <p className="my-4">There are no active learning circles right now.</p>
```

Comment:
@s-kennedy this should be marked up with ttag tags. eg:
```{t`There are no active learning circles right now.`}```

Reply:
ok thanks i updated it